posthog/hogql/property.py

```diff
@@ -170,16 +367,11 @@ def property_to_expr(
         else:
             chain = ["properties"]
 
-        if property.type == "session":
-            properties_field = None
-        else:
-            properties_field = ast.Field(chain=chain)
```

Comment:
Why is this no longer needed? (I don't actually know why it was needed in the first place, but must have been some reason)